src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -1,30 +1,54 @@
 import React, { useContext } from "react";
 import { s } from "./";
 import Link from "COMPONENTS/Link/";
-import { EpisodesContext as Episodes } from "COMPONENTS/Episodes/";
+import { 
+	ACTIONS,
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes/";
+import UTILS from "SHARED/utils.js";
 
 function renderPaginationLink(data, index){
 
 	// CONTEXT
 	// --------------------------
-	const { activeIndex } = useContext(Episodes);
+	const { 
+		state: {
+			activeIndex 
+		},
+		dispatch
+	} = useContext(Episodes);
 
 	// RENDER
 	// --------------------------
-	const { number } = data;
+	const { 
+		number: episodeNo,
+		title
+	} = data;
+
+	const episodeId = UTILS.convertToSafeString(title, "-");
 
 	const isActive = index === activeIndex;
```

Comment:
as mentioned above - make sure that 'render' is the last section before the `return` - so move the click-handlers up above it.